google-cloud-pubsub/lib/google/cloud/pubsub/subscriber.rb

```diff
@@ -284,6 +281,36 @@ def last_error
           synchronize { @last_error }
         end
 
+        ##
+        # The number of received messages to be collected by subscriber. Default is 1,000.
+        def inventory_limit
+          @inventory[:limit]
+        end
+        # @deprecated Use {inventory_limit}.
```

Comment:
I think the syntax is `{#inventory_limit}`

Reply:
Fixed.